Services/ReportsRepository/classes/class.ilObjReportBase.php

```diff
@@ -270,7 +270,7 @@ static public function getVisibleReportsObjectData(ilObjUser $user) {
 	*/
 	protected function getParentObjectOfTypeIds($type = null) {
 		$data = $this->gTree->getParentNodeData($this->getRefId());
-		while( null !== $type && $type !== $data['type'] && ROOT_FOLDER_ID !== (string)$data['ref_id'] ) {
+		while( null !== $type && $type !== $data['type'] && (string)ROOT_FOLDER_ID !== (string)$data['ref_id'] ) {
```

Comment:
Die ref_ids kommen doch aus der DB und können nur Zahlen sein. Wieso wandelst du das in einen String um für den Typenvergleich. Ist mir beim andern PR gar nicht aufgefallen.